Client: Launch Extension - cannot find module

I'm following the [Building the extension](https://github.com/streetsidesoftware/vscode-spell-checker/blob/96732fb/CONTRIBUTE.md#building-the-extension) instructions.

I start the launch config `Client: Launch Extension (Spell Checker Root)` and get:
```
Activating extension 'streetsidesoftware.code-spell-checker' failed: Cannot find module '/Users/tekumara/code/vscode-spell-checker/packages/client/dist/extension.js' Require stack: - /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js - /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js - /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js - .
```

--- a/CONTRIBUTE.md
+++ b/CONTRIBUTE.md
@@ -18,10 +18,11 @@ For the readme on the plugin: [README](./packages/client/README.md).
 
 1. `npm install -g yarn`
 2. `yarn install`
-3. Launch vscode: `code Spell Checker.code-workspace`
-4. Run the extension from vscode:
+3. `yarn build`
+4. Launch vscode: `code Spell Checker.code-workspace`
+5. Run the extension from vscode:
     1. `Debug Tab`
-    1. Choose `Launch Extension` configuration.
+    1. Choose `Client: Launch Extension (Spell Checker Root)` configuration.
     1. `F5`
 
 <sup>\*</sup> Requires Node >= 14

